docs/core/testing/unit-testing-visual-basic-with-tunit.md

```diff
@@ -0,0 +1,254 @@
+  * Configures the test runner by adding the `TUnit` package to the project file.
+
+* Add the test project to the solution file by running the following command:
+
+  ```dotnetcli
+  dotnet sln add ./PrimeService.Tests/PrimeService.Tests.vbproj
+  ```
+
+* Add the `PrimeService` class library as a dependency to the *PrimeService.Tests* project:
+
+  ```dotnetcli
+  dotnet add ./PrimeService.Tests/PrimeService.Tests.vbproj reference ./PrimeService/PrimeService.vbproj
+  ```
+
+<a name="create-test-cmd"></a>
+
+### Commands to create the solution
+
+This section summarizes all the commands in the previous section. Skip this section if you've completed the steps in the previous section.
+
+The following commands create the test solution on a Windows machine. For macOS and Unix, update the `ren` command to the OS version of `ren` to rename a file:
+
+```dotnetcli
+dotnet new sln -o unit-testing-using-dotnet-test
+cd unit-testing-using-dotnet-test
+dotnet new classlib -o PrimeService -lang VB
+ren .\PrimeService\Class1.vb PrimeService.vb
+dotnet sln add ./PrimeService/PrimeService.vbproj
+dotnet new install TUnit.Templates
+dotnet new tunit -o PrimeService.Tests -lang VB
+dotnet add ./PrimeService.Tests/PrimeService.Tests.vbproj reference ./PrimeService/PrimeService.vbproj
+dotnet sln add ./PrimeService.Tests/PrimeService.Tests.vbproj
+```
+
+Follow the instructions for "Replace the code in *PrimeService.vb* with the following code" in the previous section.
+
+## Configure Microsoft.Testing.Platform mode
+
+TUnit only supports Microsoft.Testing.Platform and doesn't support VSTest. To use `dotnet test` with TUnit, add the following configuration to your `global.json` file in the solution root:
+
+```json
+{
+    "test": {
+        "runner": "Microsoft.Testing.Platform"
+    }
+}
+```
+
+This configuration enables the Microsoft.Testing.Platform mode of `dotnet test`. For more information, see [Testing with dotnet test](unit-testing-with-dotnet-test.md).
+
+## Create a test
+
+A popular approach in test driven development (TDD) is to write a test before implementing the target code. This tutorial uses the TDD approach. The `IsPrime` method is callable, but not implemented. A test call to `IsPrime` fails. With TDD, a test is written that is known to fail. The target code is updated to make the test pass. You keep repeating this approach, writing a failing test and then updating the target code to pass.
+
+Update the *PrimeService.Tests* project:
+
+* Delete *PrimeService.Tests/UnitTest1.vb*.
+* Create a *PrimeService.Tests/PrimeService_IsPrimeShould.vb*  file.
+* Replace the code in *PrimeService_IsPrimeShould.vb* with the following code:
+
+```vb
+Imports TUnit.Assertions
+Imports TUnit.Assertions.Extensions
+Imports TUnit.Core
+
+Namespace PrimeService.Tests
+    Public Class PrimeService_IsPrimeShould
+        Private ReadOnly _primeService As Prime.Services.PrimeService
```

Comment:
Why not add `Imports Prime.Services` instead of fully qualifying?

Reply:
Fixed